utils/rpms/daos.spec

```
@@ -208,6 +208,7 @@ install -m 644 utils/systemd/%{agent_svc_name} %{?buildroot}/%{_unitdir}
 
 %pre server
 getent group daos_admins >/dev/null || groupadd -r daos_admins
+getent passwd daos_server >/dev/null || useradd -M -g daos_admins daos_server
```

Or just `daos`?
```suggestion
getent passwd daos >/dev/null || useradd -M -g daos_admins daos
```

Maybe the group should also be just `daos` also.

---

The `daos_admins` group is used to control which processes are allowed to execute the `daos_admin` privileged helper. We set the `daos_server` binary to be setgid `daos_admins` and make `daos_admin` owned by `root,daos_admins` with permissions `0750`. I think it's worth keeping this separate from the `daos` user which is intended to be used for running the DAOS servers as an unprivileged user.

---

won't "daos" user be potentially confused with any client application users with the same name in the case that the same host is used for client and server operations?

---

never mind seems this is sufficiently low likelihood to not be a concern

---

> won't "daos" user be potentially confused with any client application users with the same name in the case that the same host is used for client and server operations?

Are daos _client_ applications typically run in the context of and therefore as regular users?  That is, the client isn't a **service** on the machine but simply a binding of the client library to the end-user's process, which means it runs as that user'd ID/account?

So even on a server, if a user logs into the machine, as say, _tanabarr_ and runs an application that wants access to DAOS, it uses the daos client (i.e. libdaos) as _tanabarr_ doesn't it?